SecretSanta/test/SecretSanta.Business.Tests/GiftTests.cs

```diff
@@ -0,0 +1,70 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+
+namespace SecretSanta.Business.Tests
+{
+    [TestClass]
+    public class GiftTests
+    {
+        [TestMethod]
+        public void CreateGift_UsingValidInput_Success()
+        {
+            Gift gift = new Gift(1, "TestTitle", "TestDescription", "TestUrl", new User(1, "TestFirst", "TestLast", new List<Gift>()));
+
+            Assert.AreEqual<int>(1, gift.Id);
+            Assert.AreEqual<string>("TestTitle", gift.Title);
+            Assert.AreEqual<string>("TestDescription", gift.Description);
+            Assert.AreEqual<string>("TestUrl", gift.Url);
+            Assert.IsNotNull(gift.User);
+        }
+
+        [TestMethod]
+        [ExpectedException(typeof(ArgumentNullException))]
+        public void CreateGift_UsingNullTitle_ThrowsArgumentNullException()
+        {
+            Gift gift = new Gift(1, null!, "TestDescription", "TestUrl", new User(1, "TestFirst", "TestLast", new List<Gift>()));
```

Comment:
In these cases you can use throw away variables so the compilers will not allocate space for them.
```suggestion
            var _ = new Gift(1, null!, "TestDescription", "TestUrl", new User(1, "TestFirst", "TestLast", new List<Gift>()));
```